create-projects/create-cd-jenkins.sh

```diff
@@ -1,83 +1,110 @@
 #!/usr/bin/env bash
 set -eu
 
-# This script sets up a ${PROJECT_ID}-cd/jenkins-master and the associated webhook proxy for the project you are creating.
+# This script sets up a ${PROJECT_ID}-cd/jenkins-master and the associated
+# webhook proxy for the project you are creating.
 
 TAILOR="tailor"
-VERBOSE=false
-STATUS=false
```

Comment:
I will likely miss the ability to use this with tailor status but it was meant for development mostly so one can also locally change the script.

Reply:
I felt making the script interactive by default would be even more beneficial, as `apply` is the same as `diff` and answering `yes` to the prompt. If you still think you miss `diff`, I can change it back, too. Just thought removing it simplified the script a little, and the script being interactive made up for the loss of the functionality.

Reply:
It is ok just wanted to be sure you wouldn't miss it too after all.